### DISCUSSION TITLE
Animate website

### DISCUSSION THREAD

**Discussion by Traveler3568** [communitypoweredbydlot/community-website#70] [Ideas]
We have discussed about making the website more attractive/animated. 
We have an acquaintance who offered to help us. 

**Create an illustration to display our vision, values and mission.**

All info below. Leave your feedback in comments (specific, as requested) 

_Am facut 2 variante de ilustratie. Nici una nu e finala, dar as vrea sa stiu in ce directie sa mai lucrez._
_As vrea sa stiu (ceva de genu:)_
  _- daca va place vreuna din ele_
   _- care va place mai mult?_
  _- pros/cons la fiecare_
  _- ce ati vrea si nu este?_
 _- ce este si e bine ca este?_

_Stiu ca ati zis ca ar fi fain ceva pe gri...dar m-am gandit ca pe gri ar fi poate un pic prea monoton si de aceea am facut sa fie shades of green (am pornit de la logoul vostru de la asociatie). Mi se pare ca da o nota mai fresh si friendly. Dar putem modifica/schimba/ajusta daca nu e in ton cu ce va ganditi voi._
_La una din variante am chiar indraznit sa pun si roz. Mie imi place contrastul, dar poate ca distrage prea mult atenita._ 
_Nu am stat mult pe fiecare desen. De notat e ca fiecare personaj poate fi facut mai in detaliu, putem lucra la schema de culori mai atent, sau poate va vine o total alta idee de ilustratie si refacem tot._

Varianta 2
![Varianta 2](https://user-images.githubusercontent.com/69460413/115576600-58191900-a2cc-11eb-9b36-2544af177f1c.png)
Varianta 1 a
![Varianta 1 a](https://user-images.githubusercontent.com/69460413/115576602-58b1af80-a2cc-11eb-8da7-4f0ea755033c.png)
Varianta 1 b
![Varianta 1 b](https://user-images.githubusercontent.com/69460413/115576603-58b1af80-a2cc-11eb-8251-7feb62956a36.png)

**Comment by DanChendea** [comment]
_- daca va place vreuna din ele:_ Imi plac toate, dar cu precadere 1a si 1b

_- care va place mai mult?:_ 1b

_- pros/cons la fiecare:_ Varianta 2 - o consider potrivita pentru pagina "Implica-te", dar am anumite retineri legate de tipul acesta de design, cum ar fi: ma duce cu gandul la campania Carrefour si la logo-ul Unicef; din pacate in ziua de azi e oarecum controversat sa sugerezi unitate in rasa si gen, caci se trezeste cate un "destept" care nu se considera corect sau suficient reprezentat... dar eu consider ca sugereaza foarte bine partea de "we are one" si armonie; Varianta 1a si 1b - le consider potrivite pentru prima pagina (Misiunea). Imi place contrastul pe care il ofera rozul, asa ca prefer 1b. Daca vom merge pe aceasta varianta cu roz, sugerez sa folosim roz si in inimioara de la butonul de Doneaza, sau discret in alte locuri pe site (bullets etc.) pentru a oferii o oarecare consistenta.

_- ce ati vrea si nu este?:_ Ma refer strict la 1a si 1b - as vrea sa fie putin mai bine/clar reprezentata partea de trip/hicking, partea de camping si aninalele de companie. Chiar daca aceste activitati nu sunt explicit precizate in valori sau misiune, sunt activitati pe care le desfasuram si pe care le consider esentiale in incurajarea petrecerii timpului in aer liber. Ca exemplu (nu inseamna ca fix asa as vrea... prefer sa fiu surprins), ma gandeam la adaugarea unui personaj sau doua langa mama si copilul, eventual cu bete de trekking sau ceva sa dea de inteles ca sunt un grup intr-o drumetie. Tot aici ar merge si un catel care sa insoteasca grupul. Undeva langa el si ea intinsi pe iarba as adauga un cort, ca sa evidentiem parte de camping. 

_- ce este si e bine ca este?:_ Tot ce este in Varianta 1 este foarte bine ca este :) Imi place ca regasesc aici partea de educatie, partea de tripandpick, relaxarea in hamac/natura, musica si noua generatie :)

Abea astept sa vad o varianta finala pe site 😃  

@archblob @strugure @Traveler3568 @CIlinca

**Comment by Traveler3568** [comment]
_Am făcut un update la issue, ilustrația e pentru homepage, am cerut ceva care să reflecte Viziunea, Valorile și misiunea. 
Dacă vom dori bannere și pentru alte pagini, le vom face în concordanță cu varianta finală pentru home._ 

 **daca va place vreuna din ele**
Ambele variante 1. 

 **care va place mai mult?**
1b 

 **pros/cons la fiecare**
Varianta 2 - îmi place însă exprimă prea simplist ceea ce vrem să transmitem. Pentru homepage aș prefera ceva mai complex, cum sunt variantele 1 a/b. Conceptul într-adevăr s-ar potrivi la pagina Implică-te.

Varianta 1a - Ințial aș fi optat pentru aceasta însă cu cât mă uit mai mult la ea, cu atât mai mult simt nevoia de contrast. Parcă e prea mult verde. 

Varianta 1b - e mai plină de viață datorită roz-ului. Nu am altă propunere de culoare pentru contrast. Cum zicea și Dan, putem să  folosim culoarea și in alte elemente din site, pentru uniformitate. 

 **ce ati vrea si nu este?**
- Cort
- căței
- partea de drumeție. Aș lăsa mama și copilul așa cum sunt și aș adăuga personaje pentru drumeție, mai departe, spre munți. Ok cu propunerea lui Dan cu reprezentare prin prezența bețelor de trekking sau doar siluete mai mici, vazute de departe, fără prea multe detalii. 

 **ce este si e bine ca este?**
Super ok toate elementele. Consider fiecare din ele exprimă ceva din scopul asociației:
- natura: apa, munții/dealuri, brăduții
- petrecerea timpului în natură: relaxare, muzică, hamac, workshop
- educație/grijă față de natură: grupul de sub copac ce discută despre reciclare, personajele care străng gunoaie, mama și copilul

Mă deranjează vizual verdele din hamac pentru că nu îl regăsesc în alte elemente, eventual dacă mai facem un braduț în aceeași culoare, sau poate folosim shade-ul la adăugarea noilor elemente.

**Comment by archblob** [comment]
First of all, thank you for taking the time! it's greatly appreciated.

Second, please try to take ownership and only take our feedback as a general direction because if you try to take all that we say as something to check on a list we will end up with a mess 😄.

Third, feedback:

> - daca va place vreuna din ele
> - care va place mai mult?

**1 b**  The pink is a nice addition and the illustration fits the general spirit of the website.

> - pros/cons la fiecare

1. **a** I like the the more mild colors. There is too little color and without context it seems too busy.
1. **b** I like the the more mild colors. We have the right number of colors. Without context it seems too busy.
2. I have the same reservations that @DanChendea [mentioned](https://github.com/communitypoweredbydlot/community-website/issues/63#issuecomment-824676716) above with regards to this kind of illustration. The colors are too strong.

> - ce ati vrea si nu este?

Nothing with regards to the things illustrated, I think we are on the right track.
What I think we need going further is to also see context, where and how do you think this would fit on the website?
Fell free to reorganize any elements on the website while thinking about how what you illustrate will fit.
One thing I would consider with regards to the _busyness_ I mentioned above is braking up the illustration and sprinkling the elements on the website plane.

Pastels or watercolors.

> ce este si e bine ca este?

I like pastels and we are kind of close to that. Would not mind to go more into that direction. I like simple lines for the website and we have that too.


I'll try to find a few references and post them.

Thank you.

**Comment by ioanapastrav** [comment]
Salut,

Multumesc frumos pt feedback. Apreciez mult ce ati scris. Daca mai sunt persoane care vor scrie o sa urmaresc in continuare acest feed.
Si mie mi se pare ca se potriveste cel mai bine varianta 1b. Ma apuc sa mai lucrez la ea (inclusiv la tonurile de culori sa vedem ce functioneaza cel mai bine) si voi veni cu niste propuneri pt cum sa fie integrata pe site.

Cred ca cel mai curand o sa va trimit in saptamana de dupa sarbatori candva.

Va doresc o saptamana faina! 😊

**Comment by archblob** [comment]
Cred că deocamdată firul asta se potriveste mai bine aici.

**Comment by archblob** [comment]
Here are a few references for inspiration or to ignore 😄

http://www.species-in-pieces.com/#
https://www.sbs.com.au/mygrandmotherslingo/
https://crew.tubikstudio.com/
https://gurlic.com/
https://zavidova.com/

**Reply by ioanapastrav** [reply]
I have checked them out. I really loved some of them. I found that the styles were different from the one we began with. I am curious if you would prefer the illustration to more similar to that as in more sketchy and minimalistic (https://www.sbs.com.au/mygrandmotherslingo/) or more this style (https://crew.tubikstudio.com/) or your suggestion was regarding de site as a whole, the UX logistics and the animation part?

I loved the animations they had on each site, and I think we can make something similar for yours in the future. I am thinking something like an animated pick the trash bit (this could also be implemented on Instagram like a GIF). Or maybe something like a map of the trip and pick missions and on each place to measure the amount of trash bags collected, similar to this: 
https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.jaqjaqbird.com%2Fblogs%2Fthe-birdfeed-blog%2Fhow-to-doodle-a-cities-of-wonder-map-new-york&psig=AOvVaw3jcvBKsJRJP-3aNu2fs598&ust=1620378006819000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCMih6tLYtPACFQAAAAAdAAAAABAU

We can also animate the main illustration and by clicking each various parts of the image the user would see the mission statement related to that.

**Comment by ioanapastrav** [comment]
Hello,

I saw that you have an English speaking routine here, so I obliged.

I managed to work on the illustration and I provided few integrations for the site. You can find them below:

![Panoramic](https://user-images.githubusercontent.com/44340958/117265751-39368d00-ae5d-11eb-8fe9-a1187dfdeb3f.png)

**Var 1** 
![Var 1 Integrated_pano](https://user-images.githubusercontent.com/44340958/117265752-39cf2380-ae5d-11eb-8780-d947768f373b.png)

**Var 2**
![Var 2 Parched](https://user-images.githubusercontent.com/44340958/117265753-3a67ba00-ae5d-11eb-81c8-94aa97d5e6b8.png)

**Var 3 (slideshow/carousel)**
![Var 3 Carousel a](https://user-images.githubusercontent.com/44340958/117265754-3a67ba00-ae5d-11eb-9f38-ea974cd0bff0.png)
![Var 3 Carousel b](https://user-images.githubusercontent.com/44340958/117265743-38056000-ae5d-11eb-8368-d4e45b30d10b.png)

**Var 3 - white background**

![12Artboard 2](https://user-images.githubusercontent.com/44340958/117266675-2c666900-ae5e-11eb-881e-a95d73d7083e.png)
![13Artboard 2](https://user-images.githubusercontent.com/44340958/117266663-2a9ca580-ae5e-11eb-8052-3ca17164cde2.png)

I least prefer var 2. I find that the message is too spread out and distracting.

I prefer var 1 and 3 (with white background). I believe both of them have some advantages:

   Var 1 - breaks the column pattern with its wide display and provides interest by it's complexity and detail.

   Var 3 - provides interest by it's "movement" (the slideshow) and it is more airy than the first one, thus easier for the eye to read.


Let me know what you think!

**Reply by DanChendea** [reply]
@ioanapastrav This is great, thank you! I agree with you, in the sense that I love Var 1 and 3. I'm curious how Var 3 looks "live", so I would go with that :) Thanks again for the amazing job!

**Comment by Traveler3568** [comment]
@ioanapastrav thank you for your involvement, time and patience. 

I like all the new elements but I think there's too much pink now, with the coloured background. I liked it more on white background. 

In terms of integration, I would also exclude var 2 and all integrations on pink background.  I think that the way it will be integrated (full width illustration - var 1) / slideshow - var 3)  should be decided with @archblob as he is the expert on implementation and knows the best practices with regard to the functionality. 

@archblob please keep in mind that most people use the internet from smartphones, so our main concern should be the user experience on mobile devices.

@ioanapastrav we will discuss this today, with all members, and get back to you asap. 

Thanks again,
Hava a productive week ahead!
